src/app/AttributeCache.h

```diff
@@ -312,9 +315,16 @@ class AttributeCache : protected ReadClient::Callback
 
 private:
     using AttributeState = Variant<System::PacketBufferHandle, StatusIB>;
-    using ClusterState   = std::map<AttributeId, AttributeState>;
-    using EndpointState  = std::map<ClusterId, ClusterState>;
-    using NodeState      = std::map<EndpointId, EndpointState>;
+    // mPendingDataVersion is set when mRequestPathSet is intersecting with received attribute report
+    // mCommittedDataVersion is set when client receives the last reports and mPendingDataVersion has valid value.
+    struct ClusterState
+    {
+        std::map<AttributeId, AttributeState> mState;
```

Comment:
Maybe `mAttributes` or `mAttributeState`?

Reply:
updated it with mAttributes, thanks